modules/mesh-task/templates/consul_client_command.tpl

```diff
@@ -3,10 +3,17 @@ ECS_IPV4=$(curl -s $ECS_CONTAINER_METADATA_URI | jq -r '.Networks[0].IPv4Address
 SERVER_IP=$(cat /consul/server-ip)
 %{ endif ~}
 
+%{ if tls }
+echo "$CONSUL_CACERT" > /tmp/consul-ca-cert.pem
+%{ endif }
+
 exec consul agent \
   -advertise "$ECS_IPV4" \
   -data-dir /consul/data \
   -client 0.0.0.0 \
+  -hcl 'addresses = { dns = "127.0.0.1" }' \
+  -hcl 'addresses = { grpc = "127.0.0.1" }' \
+  -hcl 'addresses = { http = "127.0.0.1" }' \
```

Comment:
@lkysow @pglass adding these for the Consul client so that we only bind to localhost for those APIs since we don't have TLS.